modules/aleph/src/Aleph/Handler/AlephPatronHandler.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Drupal\aleph\Aleph\Handler;
+
+use Drupal\aleph\Aleph\AlephMaterial;
+use Drupal\aleph\Aleph\AlephPatron;
+use Drupal\aleph\Aleph\AlephClient;
+use Drupal\aleph\Aleph\AuthenticationResult;
+
+/**
+ * Class AlephPatronHandler.
+ *
+ * @property bool authenticated
```

Comment:
1: The property does not exist
2: Even if it did the outside is not really supposed to access it so no point in describing it as a part of the class docblock